harness/determined/common/experimental/experiment.py

```diff
@@ -16,9 +21,63 @@ class is obtained via :class:`~determined.experimental.Determined`, the
             master URL is automatically passed into this constructor.
     """
 
-    def __init__(self, experiment_id: int, master: str):
+    def __init__(
+        self,
+        experiment_id: int,
+        master: str,
+        api_ref: ExperimentsApi,
+        config: Optional[Dict] = None,
+    ):
         self.id = experiment_id
         self._master = master
+        self._experiments = api_ref
+        self.config = config
+
+    def activate(self) -> None:
+        self._experiments.determined_activate_experiment(id=self.id)
+
+    def archive(self) -> None:
+        self._experiments.determined_archive_experiment(id=self.id)
+
+    def cancel(self) -> None:
+        self._experiments.determined_cancel_experiment(id=self.id)
+
+    def delete(self) -> None:
+        self._experiments.determined_delete_experiment(id=self.id)
+
+    def kill(self) -> None:
+        self._experiments.determined_kill_experiment(id=self.id)
+
+    def pause(self) -> None:
+        self._experiments.determined_pause_experiment(id=self.id)
+
+    def unarchive(self) -> None:
+        self._experiments.determined_unarchive_experiment(id=self.id)
+
+    def wait_till_complete(self, sleep_interval: int = 5) -> None:
```

Comment:
rename to just "wait" to be closer to the name of the system call, also `docker wait`.

rename `sleep_interval` -> `interval`